src/stores/ResourceConfig/Store.ts

```diff
@@ -443,7 +446,7 @@ const getInitialState = (
     },
 
     prefillBackfilledCollections: (liveBindings, draftedBindings) => {
-        const { addBackfilledCollection } = get();
+        const { addBackfilledCollections } = get();
```

Comment:
Why are we having this stand alone function that loops through all the bindings?

We are already calling `prefillResourceConfig` right before calling this which means we loop through all the bindings twice in a row.

Reply:
Would you like to conflate the scope of `prefillResourceConfig`?

Reply:
Yeah - as best we can I would like that big initial looping to only be done once if possible.